.github/workflows/ios-testflight.yml

```
@@ -0,0 +1,392 @@
+          PROVISIONING_PROFILE_NAME: ${{ secrets.PROVISIONING_PROFILE_NAME }}
+        run: |
+          sed -i '' "s/__PROVISIONING_PROFILE_NAME__/$PROVISIONING_PROFILE_NAME/" ios/ExportOptions.plist
+          echo "ExportOptions.plist configured with profile: $PROVISIONING_PROFILE_NAME"
+
+      - name: Install certificate and profile
+        env:
+          BUILD_CERTIFICATE_BASE64: ${{ secrets.BUILD_CERTIFICATE_BASE64 }}
+          P12_PASSWORD: ${{ secrets.P12_PASSWORD }}
+          PROVISION_PROFILE_BASE64: ${{ secrets.PROVISION_PROFILE_BASE64 }}
+          KEYCHAIN_PASSWORD: ${{ secrets.KEYCHAIN_PASSWORD }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:10:50: Double quote to prevent globbing and word splitting

(shellcheck)